staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/conversion.go

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
how is `hack/update-all.sh` related to git commits? What exactly is the problem?